peachjam/views/legislation.py

```diff
@@ -187,28 +180,34 @@ def get_timeline_events(self):
                 }
             )
 
+        points_in_time = self.get_points_in_time()
+
         amendments = self.get_work_amendments()
-        if points_in_time and amendments:
+        if amendments:
             point_in_time_dates = [
                 point_in_time["date"] for point_in_time in points_in_time
             ]
-            latest_expression_date = max(point_in_time_dates)
-
-            event = [
-                {
-                    "date": amendment.get("date"),
-                    "event": "amendment",
-                    "amending_title": amendment.get("amending_title"),
-                    "amending_uri": amendment.get("amending_uri"),
-                    "unapplied_amendment": bool(
-                        amendment.get("date") not in point_in_time_dates
-                        and amendment.get("date") > latest_expression_date
-                    ),
-                }
-                for amendment in amendments
-            ]
+            latest_expression_date = (
```

Comment:
this is the key change -- if there are no points in time, then the latest date is the date of the document we're looking at (which will be a stub)